src/pip/_internal/commands/download.py

```diff
@@ -134,6 +135,13 @@ def run(self, options, args):
             reqs, check_supported_wheels=True
         )
 
+        # Download any requirements which were only fetched by metadata.
+        download_completer = PartialRequirementDownloadCompleter(
+            session,
+            progress_bar=options.progress_bar,
+            download_dir=options.download_dir)
```

Comment:
nit:

```suggestion
            download_dir=options.download_dir,
       )
```

We're gonna adopt black (albeit incrementally), so it'd be nice to follow that code style here. :)

Reply:
Absolutely :D! So glad to hear this!